src/tools/singlejar/test_util.cc

```diff
@@ -114,4 +135,19 @@ string CreateTextFile(const string& relpath, const char *contents) {
   return string("");
 }
 
+Runfiles* runfiles = nullptr;
```

Comment:
Is there a way to not use a global variable? (Maybe by initializing `runfiles` in the `setUp` method?)

Reply:
Initializing `runfiles` in the `setUp` is fine. I used global variable when I started porting tests because I did not know how much I need to call `runfiles->Rlocation`.